go/libkb/kbsig.go

```diff
@@ -137,7 +137,15 @@ func (u *User) ToTrackingStatementBasics(errp *error) *jsonw.Wrapper {
 }
 
 func (u *User) ToTrackingStatementSeqTail() *jsonw.Wrapper {
-	return u.sigs.AtKey("last")
```

Comment:
this isn't a good idea, since this dictionary arrives from the server, untrusted.